packages/dicomImageLoader/src/shared/getTypedArrayFromMinMax.ts

```diff
@@ -0,0 +1,23 @@
+export default function getTypedArrayFromMinMax(min, max) {
+  let TypedArray;
```

Comment:
This function can return undefined - please have this start with:
let pixelArrayType = Float32Array;
Also, the function name implies you are getting a typed array, but what you are getting is the type of the pixel array, so please call it something like  getPixelArrayTypeFromMinMax   I got confused looking at the function expecting it to return a typed array instance.

Reply:
There is an else statement which makes it Float32, typescript will yell if I do `let pixelArrayType = Float32Array;`